src/dataDisplay/Icon/index.tsx

```diff
@@ -78,6 +81,15 @@ const StyledIcon = styled.span<any>`
   }
 `;
 
+const StyledTooltip = withStyles(() => ({
+  tooltip: {
+    backgroundColor: theme.colors.overlay.color,
```

Comment:
so this is not customizable, right?

Reply:
yes.. but a theme level, it will use the overlay color defined by the user or the custom one if it's not overridden